backend/de.metas.material/dispo-service/src/main/java/de/metas/material/dispo/service/event/handler/ddordercandidate/DDOrderCandidateAdvisedHandler.java

```diff
@@ -146,6 +146,12 @@ protected CandidatesQuery createPreExistingCandidatesQuery(@NonNull final Abstra
 			throw new AdempiereException("supplyRequiredDescriptor shall be set for " + event);
 		}
 
+		if (CandidateType.DEMAND.equals(candidateType))
+		{
+			//This is the Advised event, there will be no MD_Candidates yet for it.
```

Comment:
pls be more explicit.
while reviewing i don't understand why....

Reply:
reworded for clarity (I hope)